src/toil/batchSystems/lsfHelper.py

```diff
@@ -140,6 +143,39 @@ def get_lsf_units(resource=False):
     else:
         return DEFAULT_LSF_UNITS
 
+def get_lsf_version():
+    """
+    Get current LSF version
+    """
+    cmd = ["bjobs", "-V"]
```

Comment:
At EBI, we get the following output for `bjobs -V`
```
$ bjobs -V
IBM Spectrum LSF 10.1.0.0 build 492718, May 25 2018
Copyright International Business Machines Corp. 1992, 2016.
US Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
```
Maybe use `lsid` instead?

```
$ lsid
IBM Spectrum LSF Standard 10.1.0.6, May 25 2018
Copyright International Business Machines Corp. 1992, 2016.
US Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
```

Note that `lsid` gives us `10.1.0.6` instead of the `10.1.0.0` from `bjobs -V`.

Confusingly, `lsid -V` gives `10.1.0.0` and a build number instead of the `0.6`!

```
$ lsid -V
IBM Spectrum LSF 10.1.0.0 build 459939, Jul 6 2017
Copyright International Business Machines Corp. 1992, 2016.
US Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.

  binary type: linux3.10-glibc2.17-x86_64
```

Reply:
oh wow, thats pretty funny that two different commands give different versions. I will switch it to lsid.